colors/nord.vim

```diff
@@ -646,6 +646,37 @@ hi! link jsNoise Delimiter
 hi! link jsPrototype Keyword
 hi! link jsRegexpString SpecialChar
 
+" TypeScript
+" > HerringtonDarkholme/yats.vim
+call s:hi("typescriptDecorator", s:nord12_gui, "", s:nord12_term, "", "NONE", "")
+hi! link typescriptOperator Operator
+hi! link typescriptBinaryOp Operator
+hi! link typescriptAssign Operator
+hi! link typescriptMember Identifier
+hi! link typescriptDOMStorageMethod Identifier
+hi! link typescriptArrowFuncArg Identifier
+hi! link typescriptGlobal Function
+hi! link typescriptBOMWindowProp Function
+hi! link typescriptClassName Function
+hi! link typescriptArrowFuncDef Function
+hi! link typescriptAliasDeclaration Function
+hi! link typescriptPredefinedType Function
+hi! link typescriptTypeReference Function
+hi! link typescriptTypeAnnotation Function
+hi! link typescriptDocNamedParamType SpecialComment
+hi! link typescriptDocNotation Keyword
+hi! link typescriptDocTags Keyword
+hi! link typescriptImport Keyword
+hi! link typescriptExport Keyword
+hi! link typescriptTry Keyword
+hi! link typescriptVariable Keyword
+hi! link typescriptBraces Normal
+hi! link typescriptObjectLabel Normal
+hi! link typescriptCall Normal
+" TypeScript JSX
+call s:hi("jsxBraces", s:nord9_gui, "", s:nord9_term, "", "", "")
+hi! link jsxTagName jsxComponentName
```

Comment:
I don't know if this actually work since the `jsxComponentName` group is not defined in this plugin and I guess the _yats.vim_ plugin uses groups prefixed with `tsx*` for React JSX syntax.

Reply:
I tried to remove 

`hi! link jsxTagName jsxComponentName`

and here is the result:

<img width="885" alt="Screenshot 2020-06-16 at 17 18 38" src="https://user-images.githubusercontent.com/3574904/84743554-7996dd80-aff5-11ea-9b44-09ac49bbdd6b.png">

Looks like 

```vim
hi! link tsxIntrinsicTagName htmlTag
hi! link tsxTagName tsxIntrinsicTagName 
```

have no effect on syntax highlighting in *.tsx files

Reply:
`hi! link jsxTagName jsxComponentName` removed